bk-api/bk_api.py

```diff
@@ -1404,9 +1350,8 @@ def handle_invalid_usage(error):
 
 
 def setup_app(app):
-
-
-
+    logging.basicConfig(level=logging.INFO,
```

Comment:
I tried to consolidate all the different places where logging was configured into a single "main" configuration here.